ocd-3441.sql

```diff
@@ -0,0 +1,2 @@
+alter table openchpl.certified_product
+add column if not exists rwt_eligibility_year integer null;
```

Comment:
If you're adding this column in here, doesn't it also need to be in the dev/openchpl_data_model.sql file, in the create script for the certified_product table?